vtex-sites/nextjs.store
# feat: Create Product Details Content component

## What's the purpose of this pull request?

This PR adds the `ProductDetailsContent` component to the starter. 

## How does it work?

The component is an accordion that contains multiple pieces of information regarding the current product.

## How to test it?

Go to a product page and find the section below the product image. You should be able to open and close the accordion items. Also, check the [JIRA Ticket](https://vtex-dev.atlassian.net/jira/software/projects/FS/boards/632?selectedIssue=FS-546) for reference (it contains a link to Figma)
## Checklist

<em>You may erase this after checking them all :wink:</em>

**PR Title and Commit Messages**
  - Available prefixes: `feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, and `test`

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Features`, `Bug fixes`, `Chore`, `Documentation`, `Style changes`, `Refactoring`, `Performance`, and `Test`
- [x] Added the component, hook, or path name in-between backticks (\`\`) - *if applicable, e.g., `ComponentName` component, `useWindowDimensions` hook*

**Dependencies**

**Documentation**
